src/target/riscv/riscv_semihosting.c

```diff
@@ -109,13 +109,13 @@ enum semihosting_result riscv_semihosting(struct target *target, int *retval)
 
 		result = riscv_get_register(target, &r0, GDB_REGNO_A0);
 		if (result != ERROR_OK) {
-			LOG_TARGET_DEBUG(target, "   -> ERROR (couldn't read a0)");
+			LOG_TARGET_ERROR(target, "   -> ERROR (couldn't read a0)");
```

Comment:
Now that these messages become more user-visible, please consider printing a full sentence. For example:

```suggestion
			LOG_TARGET_ERROR(target, "Could not read semihosting operation code (register a0)");
```

Reply:
Done